src/sdsc/__init__.py

```diff
@@ -32,50 +32,18 @@
 from lxml import etree
 from .cli import printcolor, parseargs
 from .version import __version__
+from .regex import (parentheses, sentenceends, lastsentenceends,
```

Comment:
Would `import .regex` work instead here? That would probably reduce the amount of cursing you will hear the next time I add/remove any of the regexes.